docker/exporter/exporter.py

```diff
@@ -130,12 +120,22 @@ def _export_to_file_and_zipfile(bug):
     with concurrent.futures.ThreadPoolExecutor(
         max_workers=_EXPORT_WORKERS) as executor:
       # Note: all.zip is included here
```

Comment:
```suggestion
      # Note: the individual ecosystem all.zip is included here
```

Just clarify a bit since we now have a complete all.zip as well